src/lib.rs

```diff
@@ -384,6 +390,7 @@ pub use bevy_renet::renet;
 use prelude::*;
 
 const REPLICATION_CHANNEL_ID: u8 = 0;
+const DEFAULT_EVENT_CHANNEL_RESEND_TIME: Duration = Duration::from_millis(300);
```

Comment:
I would use a little bit shorter name:
```suggestion
const DEFAULT_RESEND_TIME: Duration = Duration::from_millis(300);
```